R/sigopt_api_wrapper.R

```diff
@@ -150,25 +151,23 @@ sigopt_parse <- function(req) {
 #' @return SigOpt API token
 sigopt_api_token <- function(force = FALSE) {
   env <- Sys.getenv("SIGOPT_API_TOKEN")
-  if (!identical(env, "") && !force) return(env)
 
-  if (!interactive()) {
-    stop("Please set env var SIGOPT_API_TOKEN to your SigOpt API token",
-      call. = FALSE)
+  if (!interactive() && identical(env, "")) {
+    api_token <- SIGOPT_TEST_API_KEY
```

Comment:
this looks dangerous to me - users might accidentally fall back to the test key and not realize it